test/widgets/action_sheet_test.dart

```diff
@@ -1085,6 +1086,139 @@ void main() {
       });
     });
   });
+
+  group('channel action sheet', () {
+    late ZulipStream someChannel;
+    late PerAccountStore store;
```

Comment:
Is this needed? There's already a `late PerAccountStore store;` in an outer scope, which seems fine to reuse here.